### DISCUSSION TITLE
how can I project future dates?

### DISCUSSION THREAD

**Discussion by qn222ad** [highfestiva/finplot#251] [Q&A] [answered]
Hi,
I got a list of projected dates in the future which I want to plot on the chart so that i can know CIT, but I got this error:
must plot this primitive in prior time-range, how should I modify it?
thanks

**Comment by highfestiva** [comment, marked as answer]
It might be that you're trying to plot the second one without timestamps?

**Reply by qn222ad** [reply]
I think I got what you mean, I will try to create a fake forward date list of  price then plot the symbols of projected dates on. Thanks

[a comment by an automated account is not shown]
